.gitignore

```diff
@@ -0,0 +1,3 @@
+*.swo
+*.swp
+*.DS_Store
```

Comment:
this is going to get old quick for you :) http://islegend.com/development/setting-global-gitignore-mac-windows/